packages/react/src/wallet/useWalletLogin.ts

```diff
@@ -16,7 +16,7 @@ export function useWalletLogin() {
   const loginWallet = useWalletLoginController();
 
   return {
-    login: async (signer: Signer) => {
+    login: async (signer: Signer, profileId?: string) => {
```

Comment:
Missing the example in the `web-wagmi`. 
It seems there is also an important bug as after refreshing you still log in as a first profile and it should be the same profile you choose in the login flow (see `startWithCredentials` from `Bootstrap`)